src/test/java/steps/CheckboxSteps.java

```diff
@@ -24,48 +24,40 @@ public class CheckboxSteps {
     public CheckboxSteps(TestContext context){
         testContext = context;
         checkboxPage = testContext.getPageObjectManager().getCheckboxPage();
-        //utility = testContext.getUtility();
     }
 
 
     @Given("User is on the checkbox page")
     public void user_is_on_checkbox_page(){
-        //driver.navigate().to(configFileReader.getApplicationURL().concat("checkboxes"));
         checkboxPage.goToCheckboxPage();
 
     }
 
     @When("User clicks checkbox one {string}")
     public void user_clicks_checkbox_one(String num){
 
-        checkboxPage.clickCheckbox(checkboxPage.checkboxes, num);
-        //utility.clickAction(checkboxPage.checkboxOne);
-        //checkboxPage.clickAllCheckboxes();
+        checkboxPage.clickCheckboxByOrder(checkboxPage.checkboxes, num);
 
 
     }
 
     @And("User clicks checkbox two {string}")
     public void user_clicks_checkbox_two(String num){
-        //checkboxPage.clickCheckbox(num);
-        //utility.clickAction(checkboxPage.checkboxTwo);
-        //checkboxPage.clickCheckbox(2);
-        checkboxPage.clickCheckbox(checkboxPage.checkboxes, num);
+
+        checkboxPage.clickCheckboxByOrder(checkboxPage.checkboxes, num);
 
     }
 
-    @Then("Checkbox 1 is checked")
```

Comment:
You need to implement one step def for checking if checkbox is selected - instead of "Checkbox 1 is checked" use "Checkbox {int} is checked". Please refer to https://cucumber.io/docs/gherkin/reference/#step-arguments